WebRoot/js/zimbraMail/tasks/view/ZmTaskListView.js

```diff
@@ -482,39 +482,36 @@ function(task, colIdx) {
 	var width = (AjxEnv.isIE || AjxEnv.isSafari) ? "22" : "16";
 
 	// first row
-	htmlArr[idx++] = "<table width=100% class='TopRow'>";
+	htmlArr[idx++] = "<table class='TopRow'>";
 	htmlArr[idx++] = "<tr id='";
 	htmlArr[idx++] = DwtId.getListViewItemId(DwtId.WIDGET_ITEM_FIELD, this._view, task.id, ZmItem.F_ITEM_ROW_3PANE);
 	htmlArr[idx++] = "'>";
 
-    idx = this._getAbridgedCell(htmlArr, idx, task, ZmItem.F_SUBJECT, colIdx);
+	if (task.pComplete) {
+		htmlArr[idx++] = "<td class='ZmTaskHeaderImage'>";
```

Comment:
The selector 

`.ZmTaskHeaderImage .svg-icon,
.ZmTaskHeaderImage .progress-circle`

defines a set of properties for circular task progress bar used in detailed view.
For circular task progress bar, used in list view, all the properties are same but height and width.
That has been overridden in 
`.ZmTaskMultiView .DwtListView  .RowDouble .ZmTaskHeaderImage .svg-icon,
`.
To  reuse the existing property for circular task progress bar defined in detailed view, named class to ZmTaskHeaderImage.